src/main/java/uk/gov/pay/ledger/event/dao/EventDao.java

```diff
@@ -23,6 +24,11 @@
             " FROM event e, resource_type rt WHERE e.id = :eventId AND e.resource_type_id = rt.id")
     Optional<Event> getById(@Bind("eventId") Long eventId);
 
+    @SqlQuery("SELECT e.id, e.sqs_message_id, rt.name AS resource_type_name, e.resource_external_id, e.event_date," +
+            " e.event_type, e.event_data" +
+            " FROM event e, resource_type rt WHERE e.resource_external_id = :resourceExternalId AND e.resource_type_id = rt.id")
+    Optional<Event> getByResourceExternalId(@Bind("resourceExternalId") String resourceExternalId);
```

Comment:
I can't see it being used anywhere. Am I missing something?

Reply:
@jankowiakmaria I have removed this